app-management/guides/Editor Deep Link.md

```diff
@@ -16,51 +13,43 @@ This API generates a URL that you can use to add your app’s components to a us
+In this first case, no parameters are sent in the body of the request and your app can have any type of components. The authorization token containing your App ID is used to generate a URL that adds the app’s components to a user’s page.
 
-Consider an app that adds a lead generation form. Users create forms via the app's dashboard, and each form has a unique id. Next to each form in the dashboard the developer adds an “Add to Site'' button. It uses the Deep Link API together with the unique form id that is passed as a custom key-value pair into a Settings panel field with the same key. When users click the button, they are taken to the editor, and the component with that form id is added to the page.
+### Use Case 2: Using the Deep Link With a Custom Element
 
-## Using the API
+Parameters for the Custom Element are passed in the body of the request. They are added to the Custom Element’s Settings panel, described below, and the Custom Element is added to the page..
 
-### Authorization
+> **Note:**
+> Parameters passed in the body of the API request will only populate the Settings panel if their keys match keys already defined in the settings panel.
 
-This endpoint requires an authorization header. Pass the access token from the [OAuth installation flow](https://dev.wix.com/api/rest/getting-started/authentication).
+### Use Case 3: Using the Custom Element’s Settings Panel
 
-### Permissions
+Each Custom Element web component has its own Settings panel. Parameters may be added to the panel using the tabbed layout and a variety of UI controls that make it easy for your users.
 
-This endpoint requires the Manage Your App [permission scope](https://devforum.wix.com/en/article/available-permissions).
+![Settings panel](./../../media/custom-element-settings-panel.png "Settings panel")
 
-### Use Case 1: Adding Components
+When users install your app and a Custom ElementEditor is added to their page, they have access to the same settings panel. Values set while developing the app are presented to users as default settings.
 
-The authorization token containing your App ID is used to generate a URL that adds the app’s components to a user’s page. No body is required.
+![Custom element on page](./../../media/custom-plus-settings.png "Custom element on page")
 
-### Request
+## An Example of Using the Custom Element
 
-```CURL
-Curl -X  POST \
-https://wixapis.com/apps/v1/post-installation/editor-deep-link \
--H 'Authorization: <AUTH>'
-```
+Consider an app that adds a lead generation form. Users create forms via the app's dashboard, and each form has a unique id. Next to each form in the dashboard the developer adds an “Add to Site'' button. It uses the Deep Link API together with the unique form id that is passed as a custom key-value pair into a Settings panel field with the same key. When users click the button, they are taken to the editor, and the component with that form id is added to the page.
 
-Response
+## Editor Deep Link
```

Comment:
this section really should be in its own article

Reply:
I dont know where in the sidebar this is going (we had said the App Management section) but wont it look strange to have 2 articles about the deep link?